<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPSWidget</class>
 <widget class="QWidget" name="GPSWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPSWidget</string>
  </property>
  <widget class="QGroupBox" name="GroupBoxCom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>191</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>GPS 串口参数设置</string>
   </property>
   <widget class="QLabel" name="labelCom">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>31</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>串  口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portNameComboBoxGPS">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>33</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="baudRateComboBoxGPS">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>61</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>75</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>134</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>76800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>256000</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="dataBitsComboBoxGPS">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>89</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="parityComboBoxGPS">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>117</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>无</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>奇</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>偶</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="stopBitsComboBoxGPS">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>145</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="StartGPSbtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopGPSbtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelbaud">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeldatabits">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>89</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelparity">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>118</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>校验位</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelstopBits">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>147</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="titlelabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GPS 定位程序</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelComStatus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>串口工作状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusBar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="GroupBoxStatus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>401</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>GPS 状态数据</string>
   </property>
   <widget class="QLabel" name="labelaltit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>纬度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labellong">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>经度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeltime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelspeed">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>速度（节）：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLocationStatus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>定位状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labeldate">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>日期：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="altitudedisplay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="longtitudedisplay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="timedisplay">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="speedstatus">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="datedisplay">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="statusdisplay">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelReceived">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>GPS 接收的数据：</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textEditGPSData">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>401</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
